installer/bundle/bundle_skel.sh

```diff
@@ -412,54 +402,69 @@ install_if_program_does_not_exist_on_system()
 
 install_troubleshooter()
 {
-    echo "----- Installing troubleshooter -----"
+    # check if troubleshooter installed successfully
+    if [ ! -f "$TST_PATH" -o ! -d "$TST_MODULES_PATH" ]; then
+        # install troubleshooter if not successfully installed upon OMS install
+        echo "----- Installing troubleshooter -----"
+
+        # create temp directory
+        mkdir -p $TST_EXTRACT_DIR
+        cd $TST_EXTRACT_DIR
+
+        # grab tst bundle
+        echo "Grabbing troubleshooter bundle from Github..."
+        wget $TST_PKG > /dev/null 2>&1
```

Comment:
Kind of confused; with the latest changes, the only way that TST gets installed is via wget? Was this always the case? From what I remember, originally it was just bundled with the shell bundle, no wget involved. I think the general pattern for TST install or getting dynamic content like the .md can be try to download, but there should be a local backup.

Reply:
Sorry, I need to make it more clear in the code, but the main way is actually getting installed with the shell bundle (all the files are in `base_omsagent.data`). I tested it, and on a successful install of OMS, the TST gets installed via shell bundle, which is why I have the check to see if the paths exist just a bit above. The only time the machine should be trying to install via `wget` is if OMS installation failed in such a way that the TST didn't get copied over, in which case having the TST on the machine would hopefully help (which is why I put it in `cleanup_and_exit()`).

Reply:
Added clarification in the output + the comments in the code:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/1fcd4dd0d1cef4e18b71d295a048a3e4b7412dc9/installer/bundle/bundle_skel.sh#L404-L410